ir/json_generator.h

```diff
@@ -19,7 +19,6 @@ limitations under the License.
 
 #include <assert.h>
 #include <boost/optional.hpp>
-#include <gmpxx.h>
```

Comment:
include lib/gmputil.h for `big_int` definition.

Reply:
it's included indirectly via ir.h